src/shared/filesystemUtilities.ts

```diff
@@ -147,3 +148,16 @@ export function getNonexistentFilename(dir: string, name: string, suffix: string
         }
     }
 }
+
+/**
+ * Searches for existence of at least one file with the passed extension
+ * @param dir Directory to search
+ * @param extension Extension to look for (ex.".ts")
+ * @param exclude Pattern to ignore (ex. node_modules)
```

Comment:
the example documented here does not match the usage, which uses `**/node_modules/**`

Reply:
Removed.